db/migrate/20170809125714_add_created_at_to_awarded_point.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedAtToAwardedPoint < ActiveRecord::Migration
```

Comment:
Is there a reason why the schema isn't updated as well? Did you run this migration?